### PR TITLE
Update News.md

### PR DESCRIPTION
**Pull request by ritikakundra**
# What? Why?
Fix # .

Changes proposed in this pull request:
- a
- b

# Checks
- [ ] Runs on Heroku.
- [ ] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Follows the [Google Style Guide](https://github.com/google/styleguide).
- [ ] If this is a feature, the PR is to rc. If this is a bug fix, the PR is to master.

# Any screenshots or GIFs?
If this is a new visual feature please add a before/after screenshot or gif
here with e.g. [GifGrabber](http://www.gifgrabber.com/).

### CODE REVIEW

File: docs/News.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## June 12, 2019
+ *   **Added data**  of 1350 samples from 3 studies:
+     *  [Pheochromocytoma and Paraganglioma (TCGA, Cell 2017)](https://www.cbioportal.org/study?id=pcpg_tcga_pub) *178 samples*
```
Comment (n1zea144) [inline docs/News.md]: When do you decide to add a year to the stable study id?

File: docs/News.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+## June 12, 2019
+ *   **Added data**  of 1350 samples from 3 studies:
```
Comment (n1zea144) [inline docs/News.md]: The wording "Added data of" doesn't sound right.  In the future can you say "Added 1350 samples.." or "Added data consisting of....".

File: docs/News.md (hunk 1 above)
Comment (ritikakundra) [reply docs/News.md]: We usually do but for this case, we followed the general TCGA pub id.

File: docs/News.md (hunk 2 above)
Comment (ritikakundra) [reply docs/News.md]: Sure, will modify this statement for the future releases.

**Pull request merged**